ansible/roles/builder/tasks/main.yml

```diff
@@ -12,6 +12,61 @@
   include_role:
     name: windows-ntp
 
+- name: Install Windows-Containers feature
+  win_feature:
+    name: Containers
+    state: present
+  tags:
+    - builder_docker
+
+- name: Install feature NET-Framework-Features
+  win_feature:
+    name: 'NET-Framework-Features'
+    state: present
+  tags:
+    - builder_docker
+
+- name: Install Hyper-V feature
+  win_feature:
+    name: Hyper-V
+    include_management_tools: true
+    state: present
+  register: hyperv_installation
+  tags:
+    - builder_docker
+
+- name: Reboot the system
+  win_reboot:
+    test_command: powershell.exe 'Get-WindowsFeature'
+    pre_reboot_delay: 15
+    post_reboot_delay: 15
+  when: hyperv_installation.reboot_required
+  tags:
+    - builder_docker
+    - skip_ansible_lint
+
+- name: Wait for reconnection
+  wait_for_connection:
+  tags:
+    - builder_docker
+
+- name: Install DockerMsftProvider
+  win_psmodule:
+    name: DockerMsftProvider
+  tags:
+    - builder_docker
+
+- name: Install Docker-EE
+  win_shell: "Install-Package -Name Docker -ProviderName DockerMsftProvider -Force"
+  tags:
+    - builder_docker
```

Comment:
I meant removing this tag from everywhere

Reply:
I think we should leave the tags, so this role can be used to install Docker on other builders without modifying it.

Reply:
In that case, tag should also be on `Make sure Docker has started` task.

Reply:
But will we ever again use this tag? If it's for one time usage only then I don't think it should be commited to repo

Reply:
Probably not. Ok, I'll remove the tags.